### ISSUE TITLE
Life after Nextflow+Apache Ignite. Alternatives?

### ISSUE THREAD

**Issue by dgarrimar** [nextflow-io/nextflow#5386]
### Discussed in https://github.com/nextflow-io/nextflow/discussions/5385

<div type='discussions-op-text'>

<sup>Originally posted by **dgarrimar** October  9, 2024</sup>
Hi!

A very simple, widely used Nextflow pipeline is the following:

1) Split a large input file into chunks
2) Run a specific tool for each chunk
3) Collect the results again in the output file

The speed-up achieved increases -obviously- with the number of chunks that can run in parallel (i.e. in an HPC environment, the number of cores/cpus available).

However, in many HPC clusters (e.g. MareNostrum), the maximum number of jobs that can be running/queuing simultaneously is limited (which is reasonable, they aim to schedule large, highly parallelized jobs).
 
Until recently, I used Nextflow + Ignite to "convert" a large set of jobs (e.g. 10000 jobs using 1 CPU each) to a single job (e.g. 1 job using 10000 CPUs). Ignite was "virtualizing" a cluster using the resources and internally sending the jobs one by one (1 job/cpu). 

As Ignite is not supported anymore in NF, and working with very old NF versions that include it starts to give problems too, **I wonder whether there is an alternative in current NF versions to do this specific task.** 

Thanks a lot!

Diego

</div>

**Comment by bentsherman** [comment]
Duplicate of #5385
